README.md

```diff
@@ -15,10 +20,19 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 ## Publishing the App
 
 1. `yarn build`
-1. Login to CMS and update the css and js files generated from the build in step #1
+2. Login to CMS and update the css and js files generated from the build in step #1
 
 _Note_: Make sure that your `baseUrl` is set to the appropriate environment
 
 - Local - your web service running locally
 - Dev - testservices.bcpl.info
 - Prod - services.bcpl.info
+
+## Component Properties
+
+| Name                 | Type   | Default | Description                                                             | Is Required? |
+| -------------------- | ------ | ------- | ----------------------------------------------------------------------- | ------------ |
+| baseUrl              | string | n/a     | Api Endpoint for the Blog                                               | Yes          |
+| filters              | array  | n/a     | A list of filters to be applied to allow your users to use in their app | No           |
```

Comment:
The description for filters is a tough read. Maybe try "A list of filters that may be applied to the blog".

Reply:
Hey man, users use.